### ISSUE TITLE
Provide site url during post-install migrations

### ISSUE THREAD

**Issue by dannylamb** [Islandora/documentation#1148]
Hard lesson learned from spinning up http://future.islandora.ca

We bake in `localhost` [here](https://github.com/Islandora-Devops/claw-playbook/blob/master/post-install.yml#L43).  This led to gemini entries with `localhost` instead of `future.islandora.ca`.  We need to allow for a configurable `-l` parameter so folks can add their site name properly.

**Comment by whikloj** [comment]
Would [this](https://github.com/Islandora-Devops/claw-playbook/blob/master/inventory/vagrant/group_vars/webserver/apache.yml#L8) work or do we have (possibly) multiple vhosts?

**Comment by kstapelfeldt** [comment]
@dannylamb is this still relevant?

**Comment by dannylamb** [comment]
Yes, still relevant.  The code shifted a bit tho.  The offending line is now https://github.com/Islandora-Devops/islandora-playbook/blob/master/post-install.yml#L37
